src/main/java/ua/mateacademy/tania/patterns/adapter/AdapterPatternApp.java

```diff
@@ -0,0 +1,11 @@
+package ua.mateacademy.tania.patterns.adapter;
+
+/**
+ * Created by Таня on 06.08.2018.
+ */
+public class AdapterPatternApp {
+    public static void main(String[] args) {
+        DVIConnector dviConnector = new DVIConnectorAdapterToSVGA();
```

Comment:
You have some instance of SVGA
And must use it as DVI

Reply:
SVGAConnector ->convert to DVIConnectorAdapterToSVGA ->use as DVIConnector

Reply:
> SVGAConnector ->convert to DVIConnectorAdapterToSVGA ->use as DVIConnector

yes, i actually do this. In class lesson we've used composition. in my hw I use inharitance

Reply:
This steps illogical. You before have some object and must use it as another. But you directly create some between without start predictions